src/llama_stack_client/_client.py

```diff
@@ -182,6 +191,14 @@ def __init__(
     def qs(self) -> Querystring:
         return Querystring(array_format="comma")
 
+    @property
```

Comment:
@ehhuang how did you get these changes, this was not supposed to be here yet because adding api_key Stainless config was only in a dev branch but not in main. It's ok if we leave this here though we will just need to remove api key changes from the sync_stainless script.